_faqs/faq-12.md

```diff
@@ -1,8 +1,110 @@
+Members in the community mostly belong to one (or more) of the following groups: academia, industry, and the makers/hobbyists.
+
+RIOT is distributed under a copy-left license and strongly prefers free and open source components over proprietary code and binary blobs.
+Still, RIOT does contain proprietary drivers (some even using binary blobs) where no free alternative is available, e.g. for the ESP WiFi chipset.
+<!-- FIXME: Make FAQ entries linkable and link to "Why LGPL?" -->
+
+## Vendor Specific SDKs
+
+Most MCU vendors provide an SDK that is tied and only supports MCU of that specific vendor.
+The advantage of the vendor SDKs is that they are developed in lock-step with the hardware and expose all features of the hardware early on.
+The RIOT community can only start working on drivers once the MCUs are generally available and more exotic features that are only relevant on niche use cases may never be supported in favor of a cleaner and more portable API in RIOT.
+
+From a government point of view RIOT is developed by a self-governed community, while the vendor SDKs are under tight control of the vendor.
+The RIOT community has so for maintained many popular boards well after the vendors declared them obsolete, while vendor SDKs often phase out support for older MCUs sooner.
+
+Vendor specific SDKs typically do not have any copyleft clauses.
+
+## Arduino
+
+Both RIOT and Arduino have a strong focus on a gentle learning curve and a low entry barrier.
+Arduino however goes a step further and sacrifices advanced features such as native multi-threading, a full-fledged and mature network stack, or VFS subsystem for simplicity.
+This combined with a trimmed down IDE results in a lower entry barrier for Arduino compared to RIOT.
+
+RIOT still allows new users to quickly get productive, but without limiting developers to specific design patterns, specific software architectures, and a strong tie to a specific IDE.
+As a nice treat for users that started with Arduino but ran in limitations, our [Arduino compatibility layer][riot-arduino] allows re-using sketches and libraries from the Arduino world.
+
+Arduino has a vast ecosystem of libraries developed independently by 3rd parties.
+RIOT set of modules and packages is not that vast, but RIOT's modules are seamlessly integrated, have a consistent APIs and integrate well with each others.
+This results in easier reuse of code and higher productivity without having to worry about interoperability issues of components and "impedance matching" between misaligned APIs.
+
+The Arduino software is released under a copyleft license and hardware support can be extended greatly beyond the official Arduino boards by using 3rd party Arduino cores.
+The development of the ecosystem is sponsored and controlled by a single company that owns the registered trademark of Arduino, while RIOT governed by an independent community.
```

Comment:
last part of the sentence, maybe change it into:
- 'while with RIOT an independent community is responsible for governance.' 
or
- 'while RIOT applies self-governance by its independent community.'

Goal of the change is again avoiding 'government' and using 'governance' instead. Not sure about the exact wording here.

Reply:
Thx, addressed by 9ea4bc1bf1609ef5dfbea27b6fc798b653180b17